phpBB/includes/functions.php

```diff
@@ -5448,22 +5436,6 @@ function phpbb_user_session_handler()
 }
 
 /**
-* Check if PCRE has UTF-8 support
-* PHP may not be linked with the bundled PCRE lib and instead with an older version
-*
-* @return bool	Returns true if PCRE (the regular expressions library) supports UTF-8 encoding
-*/
-function phpbb_pcre_utf8_support()
```

Comment:
It looks like we could easily add a dummy function returning true to functions_compatibility.php. @s9e What do you think?